### PR TITLE
TabPanel: Add iconPosition option

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

**Review by EugeniyKiyashko — DISMISSED** [review]

**Review by Zedwag — DISMISSED** [review]

**Review by vladaskorohodova — DISMISSED** [review]

File: packages/devextreme/js/ui/tab_panel.d.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ import {
 
 import {
     Position,
+    TabsIconPosition,
```
Comment (IlyaKhD) [inline packages/devextreme/js/ui/tab_panel.d.ts:21]: Add reexport:

```ts
export {
    TabsIconPosition,
} from '../common';
```

Comment (marker-dao) [reply packages/devextreme/js/ui/tab_panel.d.ts:21]: I'm not sure that we should export this type separately. We export the entire dxTabPanelOptions interface.

Comment (IlyaKhD) [reply packages/devextreme/js/ui/tab_panel.d.ts:21]: It's the rule for all components - all public types should be reexported

Comment (marker-dao) [reply packages/devextreme/js/ui/tab_panel.d.ts:21]: Was updated here [e20fbd7cff57e643bde9fe631f671be9dbb154a4].

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**